Automattic/wp-calypso
# Help Center: Add contact support option for article negative feedback

Fixes: https://github.com/Automattic/wp-calypso/issues/86531

## Proposed Changes

* Display "contact support" button for article feedback forms. This will help align the behavior of the feedback forms. We will revisit how to collect qualitative feedback later on.

The contact support button disabled styling should also be fixed now. The button colors (icon and text) will have a gray color when disabled.

**Article Feedback:**
<img width="411" alt="CleanShot 2024-07-11 at 13 22 12@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/68ad4ca1-5806-44f0-8772-563200668455">


## Why are these changes being made?


* Align Help Center feedback form behavior.

## Testing Instructions



* Checkout branch
* Open the Help Center again, click on a random article such `Domain`, scroll to the bottom of the article and click on the thumbs down feedback icon.
* Verify the `Contact WordPress.com Support` button appears and open the Zendesk support chat when clicked.
* Open the Help Center, Click `Still need Help?`, send a message, and click the thumbs-down feedback icon. 
* Verify the `Contact WordPress.com Support` button appears and open the Zendesk support chat when clicked.
* Force the `renderEmailOption` component to appear and test the contact email option ( you can force the value to be true `packages/help-center/src/components/help-center-contact-page.tsx` - line 162 ).
* Verify the `calypso_helpcenter_feedback_contact_support` tracks event is triggered after clicking the `Contact WordPress.com Support` button from an article.
* Verify the `calypso_odie_extra_contact_option` tracks event is triggered after clicking the `Contact WordPress.com Support` and email support buttons from `Still need help?`.
* Verify the `wpcom/v2/help/zendesk/update-user-fields` request includes an initial message when support contact is triggered from the article feedback form (The initial message should consist of the article URL).

## Pre-merge Checklist
